### ISSUE TITLE
Problems with accents in subtitles.LG49UB8500 Brazil.

### ISSUE THREAD

**Issue by jrstravino** [simonjhogan/PLAYz-Media-for-PLEX#103]
I bought the application by LG the store, but the subtitles are cut where have accents in words.
Many people who want to use the playz-media are with the same problems.
This seems like a simple solution, how can we solve?
sorry for my bad english

**Comment by marcosharbs** [comment]
Podes abrir o arquivo de legenda no Notepad++ e converter para UTF-8 que a acentuação irá funcionar. Também passo por isso com todas as legendas quase.

**Comment by jrstravino** [comment]
Sim o desenvolvedor precisa arrumar isto, mas temos novidades, em breve vai sair a versão oficial do cliente Plex para as tvs Lg disponível gratuitamente.

Date: Fri, 27 Mar 2015 04:58:25 -0700

**Comment by braulioholtz** [comment]
Na realidade acredito que a falha possa ser da própria LG, até antes para mim cortava até onde tinha acento tipo "você é legal" apareceria apenas "voc", anteontem atualizei e quando tem acento, não aparece, dai só aparece a legenda novamente quando a frase não tem acento. Dei uma olhada rápida nas documentações e o código e na realidade o código (e seguindo a documentação) apenas linka direto na tag de video o caminho da legenda. Agora, para mim no WebOS pelo menos, o HLS funciona normalmente (acho que é devido ao PC fazer o processamento, vou pesquisar melhor), funciona a legenda, mas não consigo avançar ou retroceder

**Comment by jrstravino** [comment]
Não acredito que seja problema da LG pois as legendas funcionam perfeitamente com os acentos diretamente pela TV e também pela rede através do DLNA.O Playz tem problema com o unicode, se você salvar as legendas com a codificação UTF-8 elas funcionam corretamente no playz-media.É uma pena que ainda não liberaram o aplicativo oficial do Plex.

Date: Fri, 31 Jul 2015 06:02:02 -0700

**Comment by braulioholtz** [comment]
Sim, até entendo, mas o app usa HTML5, tag video, de alguma forma linkar a legenda dá errado, já por DLNA e pela TV deve usar um SDK próprio (HLS, ou Widevine).
O que eu entendi, é que se nas opções não selecionar o HLS, a legenda fica incorreto, se não for pelo HLS, usa o método URI http://developer.lge.com/webOSTV/api/supported-standard-web-apis/html5-standard/audio-video/mediaoption-parameter/ Media Transport Types
No método normal, não há nada no código que encontrei que crie uma legenda por cima (como uma div) mas sim por algum processo interno, é utilizado uma mediaOption usando o option.subtitle, isso fica no arquivo player.js https://github.com/simonjhogan/PLAYz-Media-for-PLEX/blob/master/WebContent/system/script/player.js#L943 linha 943
Existe outro app chamado xplay, ainda não testei ao certo, mas me desanimei porque é uma demo de 10 dias e não consigo controlar por um controle remoto comum, tem que ser o magic eu acredito, ou o celular.
Então, a ultima atualização a legenda com acento não aparece, a frase toda, antes da última atualização aparecia até antes do acento.
Modo HLS funciona a legenda, mas existe um problema em avançar ou atrasar (talvez a solução mais fácil de corrigir).
Vou tentar reescrever o código, com base no trabalho do Simon, já fiz algumas mudanças no repositório e é aberto para o Simon usar

**Comment by jrhames** [comment]
Folks, try to keep this english only. This might be relevant for non Portuguese speakers as well. Obrigado :)

**Comment by marcosharbs** [comment]
O problema da acentuação da legenda ta no response do servidor e não no App. Eu reescrevi e usei uma lib javascript que cria um loop e fica renderizando a legenda numa div por cima.

**Comment by braulioholtz** [comment]
Cool Marcos, I have made some adjustments in my repository but are more visual adjustments (some how to get the play / pause button appearing at the same time, small icons in full HD screens).
I've been searching to develop using the Moonstone and of course I will share the code, but I'm racking my brain with the development at the top, I wanted to use as the content store lg
Mark, the problem of not running the accent would be Plesk Media server again? In the app I tried a solution, but there seemed to be a simple solution, because the video tag code is simple in this matter, I thought that could be the problem WebOS, but I did tests getting another file (not Plesk response).
Plesk there is this work of Simon (and forks) and another XPlay, but that other is paid and the developer did not share the code.

**Comment by danielcbit** [comment]
Hello everybody.
I'm a little confused here. So the problem is with PMS, with PLayZ or with WebOS?

I'm trying to watch a 3D movie with .ass subtitle, but as braulioholtz noted, only texts that do not have accents are rendered.
This happens even when running from USB for example.
